website/pages/en/index.js

```diff
@@ -233,8 +233,8 @@ class Index extends React.Component {
           <div className="split-right">
```

Comment:
Can you quote the `'` in don't ?

Reply:
Can you add the sbt command in the Travis terminal window ?

Reply:
This is a syntax highlighting bug in GitHub, "don't" renders fine

<img width="585" alt="screen shot 2018-08-31 at 16 13 07" src="https://user-images.githubusercontent.com/1408093/44917516-d4841580-ad38-11e8-8e1f-dfab2a624d75.png">

> Can you add the sbt command in the Travis terminal window ?

There's not enough vertical space to fit another line, I tried tweaking the css but couldn't figure it out so that the diff doesn't get hidden under the travis bar.
<img width="1197" alt="screen shot 2018-08-31 at 16 14 05" src="https://user-images.githubusercontent.com/1408093/44917560-fb424c00-ad38-11e8-8c2e-957d43e9fa36.png">